24.1 DS configuration docs updates (#91)
seqeralabs/docs: platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

    ```

1. Open `data-studios.env` in an editor, and make the following changes:

    1. Uncomment the `connect-proxy` and `connect-server` services.
    1. Set the following environment variables:

>         - `PLATFORM_URL`: The same value assigned to `TOWER_SERVER_URL`.

Author: ```suggestion
        - `PLATFORM_URL`: The same value assigned to `TOWER_SERVER_URL`. For example, `https://example.com`.
```

>         - `CONNECT_PROXY_URL`: A base domain name for the connect proxy subdomain. For example, if you specify the base domain `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.

Author: ```suggestion
        - `CONNECT_PROXY_URL`: A URL for the connect proxy subdomain. We recommend you set a first-level subdomain of your `PLATFORM_URL` for your connect proxy. For example, `https://connect.example.com`.
```

    client_token=$(openssl rand -base64 32 | tr -d /=+ | cut -c -32)
    ```

1. Edit the `server.yml` file and set the `CONNECT_REDIS_ADDRESS` environment variable to the hostname or IP address of the Redis server configured for Platform.

1. Edit the `proxy.yml` file and set the following variables:

    - `CONNECT_REDIS_ADDRESS`: The hostname or IP address of the Redis server configured for Platform.

>     - `CONNECT_PROXY_URL`: A base domain URL for the connect proxy subdomain. We recommend you set the base URL for your Platform installation (`PLATFORM_URL` below) for your connect proxy, to be able to use the same wildcard TLS certificate for all data studio instance URLs and avoid additional domain nesting. 

Author: ```suggestion
    - `CONNECT_PROXY_URL`: A URL for the connect proxy subdomain. We recommend you set a first-level subdomain of your Platform installation domain (`PLATFORM_URL` below) for your connect proxy, to be able to use the same wildcard TLS certificate for all data studio instance URLs and avoid additional domain nesting. For example, `https://connect.example.com`.
```

    - `PLATFORM_URL`: The base URL for your Platform installation, such as `https://example.com/`.

1. Edit your Platform installation's `ingress.eks.yml` file:

    - Uncomment the `host` section at the bottom of the file.

>     - Replace `<CONNECT_DOMAIN>` with the domain of your `CONNECT_PROXY_URL` value specified in the `proxy.yml` file above. For example, if your connect proxy URL is `https://example.com`, set the `<CONNECT_DOMAIN>` value as `example.com`.

Author: ```suggestion
    - Replace `<YOUR-TOWER-HOST-NAME>` with the base domain of your Platform installation. For example, `example.com`.
```
